MastodonIntent/Handler/SendPostIntentHandler.swift

```diff
@@ -97,6 +97,7 @@ extension SendPostIntentHandler: SendPostIntentHandling {
                     subtitle: content,
                     image: response.value.account.avatarImageURL().flatMap { INImage(url: $0) }
                 )
+                post.url = URL(string: response.value.url ?? "")
```

Comment:
Maybe it makes sense to use an `if let` here, if `response.value.url` is invalid and not add the URL instead of adding a URL with an empty string? I don't have any hard feelings on this, just an idea :)

Reply:
That’s definitely better, I used the fact that `URL(string: "")` returns nil here but that’s not very understandable (and it definitely took me a second to remember that’s what happens when looking at the code again). Going with your approach 👍

Reply:
Oh, you're absolutely right, I didn't know that, so thank you :)